core/src/main/java/tc/oc/pgm/listeners/PGMListener.java

```diff
@@ -258,6 +260,26 @@ public void handleItemPickup(final PlayerPickupItemEvent event) {
     if (nearestPlayer != event.getPlayer()) event.setCancelled(true);
   }
 
+  @EventHandler
+  public void lockFireTick(final MatchLoadEvent event) {
+    event.getMatch().getWorld().setGameRuleValue(DO_FIRE_TICK, Boolean.toString(false));
+  }
+
+  // unlock firespread if not disabled by <gamerule>
+  @EventHandler
+  public void unlockFireTick(final MatchStartEvent event) {
+    ImmutableMap<String, String> lockFireTick =
+        event.getMatch().getModule(GameRulesMatchModule.class).getGameRules();
+    if (lockFireTick.isEmpty() || Boolean.parseBoolean(lockFireTick.get(DO_FIRE_TICK))) {
+      event.getMatch().getWorld().setGameRuleValue(DO_FIRE_TICK, Boolean.toString(true));
```

Comment:
You're setting it to TRUE if it's defined in `<gamerules>` but what if it's not defined at all? Currently you're breaking any map that defines this in the world data and not in xml (older maps)